TrafficCapture/coreUtilities/src/main/java/org/opensearch/migrations/coreutils/MetricsEvent.java

```diff
@@ -0,0 +1,30 @@
+package org.opensearch.migrations.coreutils;
+
+public enum MetricsEvent {
```

Comment:
I was thinking about this, because grouping by application makes a lot of sense. I won't have this in the about-to-be-pushed update, but I think it makes sense, and I'm looking into the link you included--thanks.